src/Playroom/Playroom.tsx

```diff
@@ -59,7 +61,7 @@ const getTitle = (title: string | undefined) => {
 };
 
 export interface PlayroomProps {
-  components: Record<string, ComponentType>;
+  components: Record<string, ComponentType<any>>;
```

Comment:
`ComponentType` defaults to a component with no props, specifying `any` here permits components with or without props, which is obviously more accurate than before.